orcid-web/src/main/java/org/orcid/frontend/web/controllers/PasswordResetController.java

```diff
@@ -297,10 +297,15 @@ private boolean isTokenExpired(PasswordResetToken passwordResetToken) {
 
     @RequestMapping(value = "/sendReactivation.json", method = RequestMethod.POST, produces = MediaType.APPLICATION_JSON)
     public ResponseEntity<?> sendReactivation(@RequestParam("email") String email) throws UnsupportedEncodingException {
+        
+        if (!email.contains("@")) {
+          email = URLDecoder.decode(email, "UTF-8");  
+        } 
+        
         email = OrcidStringUtils.filterEmailAddress(email);
         String orcid = null;
         
-        if (!email.contains("@")) {
+        if (!OrcidStringUtils.isEmailValid(email)) {
```

Comment:
@DanielPalafox to validate email addresses on registration, we use the `org.apache.commons.validator.routines.EmailValidator`, please see the example here: https://github.com/ORCID/ORCID-Source/blob/master/orcid-web/src/main/java/org/orcid/frontend/web/controllers/BaseController.java#L293

Lets use the same logic for the reset password, so, we get consistent results.

Thanks

Reply:
Ok